app/mailers/user_info_changed_mailer.rb

```diff
@@ -1,9 +1,10 @@
 class UserInfoChangedMailer < ApplicationMailer
   layout false
 
-  def user_info_changed(user, info)
+  def user_info_changed(user, info, previous_email=nil)
     @user = user
-    @email_to = user.email
+    @recipient_emails = [user.email]
+    @recipient_emails << previous_email if previous_email.present? && info == 'email'
```

Comment:
`previous_email.present?` can be shortened to just "if previous_email"

Also I think this line can be moved to inside the `when "email"` block in the case statement below as they both have the same condition.